ts/src/base/Exchange.ts

```diff
@@ -3957,6 +3957,14 @@ export default class Exchange {
         return [ value, params ];
     }
 
+    handleParamString2 (params: object, paramName1: string, paramName2: string, defaultValue: Str = undefined): [string, object] {
```

Comment:
first we can't have this return type and second, this is not consistent with the `handleOptionAndParams2` (here the 2 means 2 different params, whereas there it represents 2 different methods), should we update the other implementation?

Reply:
return type - can you plz tell wdym about return type? the regular `handleParamString` (without 2) has that same return type

Reply:
i think that using `2` in the end of the methods across mostly means two have to provide two keys of the provided dict, but `handleOptionAndParams2` accepts two methods, yeah. as you decide, imo `handleParam2` doesn't include any method-specific parts, so `2` there might be intuitive that it accepts two keys from `params` dict. we might not touch `handleOptionAndParams2` and leave it as is. 
as you tell me

Reply:
@carlosmiei can you comment on this ^